### PR TITLE
PMCP-3765 & PMCP-3766: You may also like & PMC most popular module

### PR DESCRIPTION
**Pull request by saniyat**
* larva-patterns - Add `brand-most-popular` module.
* larva-patterns - Update `newswire` module and add support for slider.
* larva-css - Add `a-grid` classes.
* larva-css - Add `lrv-u-font-size-16@mobile-max` class
* larva-css - Add `.lrv-u-border-l-00\@mobile-max`, `.lrv-u-border-r-00\@mobile-max`, `.lrv-u-border-t-1\@mobile-max` & `.lrv-u-border-b-00\@mobile-max` classes.

### Make sure you complete these items:

- [x] Updated root CHANGELOG.md with summary of changes under `Unpublished` section
- [x] `npm run prod` in this repo outputs expected changes (excepting the issue with re-ordered partials in larva-css algorithms partials - see [LRVA-1885](https://jira.pmcdev.io/browse/LRVA-1885))
- [x] If adding a new pattern, in the PR comment, included a screenshot and link to the static Vercel deployment
- [ ] If changes to build scripts or the Node.js server, tested changes in pmc-spark [via a pre-release](https://confluence.pmcdev.io/x/XhOeAw)
- - If changes to build tools: npm scripts `prod`, `lint`, and `dev` scripts run as expected
- - If changes to Larva server: Static site generates as expected in a theme  (avail. on a URL {brand}.stg.larva.pmcdev.io)

### CODE REVIEW

**Comment by akashdhawade2005** [comment]
New Modules Added:
1. Brand most popular: https://pmc-larva-git-feature-pmcp-3765-penske-media-corp.vercel.app/larva/modules/brands-most-popular/
2. Newswire module changes to support for slider: https://pmc-larva-git-feature-pmcp-3765-penske-media-corp.vercel.app/larva/modules/newswire/
Screenshot at Apr 08 19-55-46
Screenshot at Apr 08 19-56-03

File: packages/larva-css/src/03-utilities/u-border.common.inline.scss [hunk 1]
```diff
@@ -55,6 +55,12 @@
 	}
 }
 
+.lrv-u-border-t-3\@desktop {
```
Comment (laras126) [inline packages/larva-css/src/03-utilities/u-border.common.inline.scss:58]: This can be added in the breakpoint grouping below

**Review by laras126 — COMMENTED** [review]
One small change, otherwise this looks great!

File: packages/larva-css/src/03-utilities/u-border.common.inline.scss (hunk 1 above)
Comment (akashdhawade2005) [reply packages/larva-css/src/03-utilities/u-border.common.inline.scss]: @laras126 Done.

**Review by laras126 — CHANGES_REQUESTED** [review]
A few additional changes requested - thanks!

File: packages/larva-patterns/modules/newswire/newswire.prototype.js [hunk 2]
```diff
@@ -32,8 +34,12 @@ o_card.c_lazy_image.c_lazy_image_crop_class = 'lrv-a-crop-4x3';
 o_card.c_span = c_span;
 o_card.c_title = c_title;
 o_card.c_timestamp = c_timestamp;
+o_card.c_svg = c_svg;
 
 module.exports = {
+	newswire_classes: 'lrv-a-wrapper lrv-u-padding-b-2',
+	newswire_list_classes: 'lrv-a-unstyle-list js-Flickity js-Flickity--profile js-Flickity--isContained js-Flickity--nav-top-right js-Flickity--bordered-buttons js-Flickity--isFreeScrol js-Flickity--newswire lrv-u-padding-l-00',
```
[the remark this answers is not in the document]
Comment (akashdhawade2005) [reply packages/larva-patterns/modules/newswire/newswire.prototype.js]: @laras126 You were right js-Flickity--profile was not need for the pattern, instead I use js-Flickity-fourths

**Review by laras126 — APPROVED** [review]
Looks great, thanks! One last thing - was this change checked on Indiewire? That theme might be using the newswire template from pmc-plugins/pmc-larva in the footer-larva.php partial (or similar name to that). We need to confirm this does not cause a breaking change before releasing it.

**Comment by akashdhawade2005** [comment]
> Looks great, thanks! One last thing - was this change checked on Indiewire? That theme might be using the newswire template from pmc-plugins/pmc-larva in the footer-larva.php partial (or similar name to that). We need to confirm this does not cause a breaking change before releasing it.

@laras126 I believe you confirmed that Indiewire is using the template and as per @asannad  we need to move the old one to Indiewire. Please see this thread: https://penskemediacorp.slack.com/archives/C02TE02C8UB/p1649264886595449

**Pull request merged**